test.js

```diff
@@ -281,10 +281,30 @@ test("handles escaped double-quotes in double-quote strings", function(t) {
     source: 'abc "a\\"bc" foo cba',
     target: "c",
   }), [ 2, 16 ]);
+  t.deepEqual(styleSearchResults({
+    source: 'content: \\"\n',
```

Comment:
I find this and some following tests kind of confusing because there is no string to check, or else the string is not closed. Would it clarify things to change these tests so they always include a complete string?

Reply:
I think this PR was actually a little misguided, so I'm going to go ahead and close it. Thank you!